base/ltfssaxes.dtx

```diff
@@ -132,8 +132,9 @@
 % The rules set up use explicit series values not \cs{..default}
 % indirections; my current feeling is that this is in fact better.
 %
-% With 9 weights and 9 width classes this table is getting a bit large
-% in the end (324 entries) but on the other hand it doesn't change and
+% With 9 weight values (from \texttt{ul} to \texttt{ub}) and 9 width values
+% (from \texttt{uc} to \texttt{ux}), this table is getting a bit large in
+% the end (1345 entries), but on the other hand it doesn't change and
 % accessing speed and it is fast this way.
```

Comment:
An incomplete sentence: "but on the other hand it doesn't change and accessing speed and it is fast this way." What was that originally supposed to mean?

Reply:
@user227621  

I assume the intended meaning is along the lines of this, which might be clearer?

this table is now rather large  (1345 entries), but on the other hand, the table doesn't change and accessing rules is fast using a table implemented in this way.

Reply:
I have rephrased the sentence accordingly.